quantum-viz/pyproject.toml

```diff
@@ -24,6 +24,7 @@ python = "^3.6.1"
 click = "^8.0.1"
 ipython = "<8.0.0"
 varname = "^0.8.1"
+qiskit-terra = {extras = ["qiskit"], version = "^0.18.3"}
```

Comment:
When building the package with `poetry build` and installing the `.whl` file with `pip` there's a warning:
```
  WARNING: qiskit-terra 0.18.3 does not provide the extra 'qiskit'
```
We should address that when `qiskit-terra` is returned to the dev dependencies (so that it will be an optional dependency).

Reply:
Done